src/components/MovieCard.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class MovieCard extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    const movieName = this.props.movieName ? this.props.movieName : 'name';
```

Comment:
Think about using Nullish coalescing operator (??) instead of Ternary operator.

Reply:
Learned and used operator (??).